addons/website/models/ir_ui_view.py

```diff
@@ -469,6 +469,10 @@ def _prepare_qcontext(self):
                 editable=editable,
             ))
 
+            # fetch non-prefetchable fields of the mixin SeoMetadata, as they
+            # will be read later in non-sudo mode
+            self.sudo().read(['website_meta_title', 'website_meta_description', 'website_meta_keywords'])
```

Comment:
Shouldn't we do that where we are already pretching views and pages here https://github.com/odoo/odoo/blame/07c45ed7bd305b89ae616d7344c6c96d9af63f1b/addons/website/models/ir_http.py#L267 ?

Reply:
Well, we remove the prefetch for `['website_meta_title', 'website_meta_description', 'website_meta_keywords']`, then no. Or I don't understand your point ?